symfit/core/argument.py

```diff
@@ -46,6 +47,15 @@ def __init__(self, name=None, *args, **assumptions):
             self.name = name
         super(Argument, self).__init__()
 
+    def __getstate__(self):
```

Comment:
this is the same code as in your seperate PR for pickling. Git is difficult? :P

Reply:
Bit of both. I need it to do the hyperparameter optimization (using Pygmo), which does multiprocessing, and therefore needs to pickle things.Just merge #158 first, and all is peachy.